src/framework/standard/structures/buckets.rs

```diff
@@ -469,7 +469,53 @@ impl BucketBuilder {
         self
     }
 
-    /// This function will be called once a user's invocation has been delayed.
+    /// This function is called when a user's command invocation is delayed when:
+    /// 1. await_ratelimits is set to a non zero value (the default is 0).
+    /// 2. user's message rests comfortably within await_ratelimits (ex. if you set it to 1 then it will only respond once when the delay is first exceeded).
```

Comment:
```suggestion
    /// This function is called when a user's command invocation is delayed when:
    /// 1. `await_ratelimits` is set to a non zero value (the default is 0).
    /// 2. user's message rests comfortably within `await_ratelimits` (ex. if you set it to 1 then it will only respond once when the delay is first exceeded).
    ///
    /// You can use this to, for example, send a custom response when someone exceeds the amount of commands they're allowed to make.
```